### PR TITLE
Fix TX/RX charts. Add UI_CHART_TYPE

### PR DESCRIPTION
**Pull request by pheiduck**
From @suxscribe 
Fixes: #859.

Also adds `UI_CHART_TYPE` option to enable and set chart type:

```
UI_CHART_TYPE=0 # Charts disabled
UI_CHART_TYPE=1 # Line chart
UI_CHART_TYPE=2 # Area chart
UI_CHART_TYPE=3 # Bar chart
```

Needs some testing in terms of performance. On my PC it works fine and only adds about 7-10% CPU and 3-5% GPU load.

<details>
  <summary>Chart screenshots:</summary>

  Bar

  Area

  Line

</details>

### CODE REVIEW

**Comment by peterlewis** [comment]
Looks good! Just note the one comment left... :)

File: src/www/js/app.js [hunk 1]
```diff
@@ -23,14 +23,33 @@ function bytes(bytes, decimals, kib, maxunit) {
   return `${parseFloat((bytes / Math.pow(k, i)).toFixed(dm))} ${sizes[i]}`;
 }
 
+const darkModeMediaQuery = window.matchMedia('(prefers-color-scheme: dark)');
```
Comment (peterlewis) [inline src/www/js/app.js:26]: Just checking this section aligns to our use of `media` for automatic dark/light mode?

Comment (pheiduck) [reply src/www/js/app.js:26]: I guess this is not needed as we defined media auto light dark here: https://github.com/wg-easy/wg-easy/blob/master/src/tailwind.config.js

Comment (suxscribe) [reply src/www/js/app.js:26]: I didn't quite understand how it works currently :)
The added query is needed to define colors of the charts for light / dark mode.

I can't find where `toggleTheme` method is used, and nothing happens when setting `theme` value in local storage to either `light` or `dark`.

There probably should be a toggle in the UI. The media query could be used to get current browser theme and then switch it with the toggle if needed.

**Comment by pheiduck** [comment]
Auto light/dark mode is working for charts.

**Review by suxscribe — APPROVED** [review]
Looks good

**Comment by suxscribe** [comment]
I've made a few updates - added chart and dark/light mode toggle buttons.
Should I commit them in this PR or make a new one?

**Comment by pheiduck** [comment]
@suxscribe sure, if it does not break the auto light/dark mode switch

**Comment by peterlewis** [comment]
@suxscribe before you make a new PR it's worth pointing out that the underlying mechanism and code for light/dark toggle still exists from the merge upstream from fork, we just ripped out the toggle and switched to using `media`. See the following:

https://github.com/pheiduck/wg-easy/commit/de9eebd82703faa4d24bcbedce4e8118897e5ac1

**Comment by pheiduck** [comment]
> @suxscribe before you make a new PR it's worth pointing out that the underlying mechanism and code for light/dark toggle still exists from the merge upstream from fork, we just ripped out the toggle and switched to using `media`. See the following:
> 
> [pheiduck@de9eebd](https://github.com/pheiduck/wg-easy/commit/de9eebd82703faa4d24bcbedce4e8118897e5ac1)

Yes I perfer the new methode anyway.

**Comment by suxscribe** [comment]
@peterlewis Oh okay, I've changed that to `selector` =) . And added ability to switch between auto dark mode and forcing light or dark

**Pull request merged**